openshift/tests-extension/test/qe/specs/olmv1_cc.go

```diff
@@ -34,4 +43,1061 @@ var _ = g.Describe("[sig-olmv1][Jira:OLM] clustercatalog", g.Label("NonHyperShif
 
 	})
 
+	g.It("PolarionID:69123-[Skipped:Disconnected]Catalogd clustercatalog offer the operator content through http server", g.Label("LEVEL0"), func() {
+		var (
+			baseDir                = exutil.FixturePath("testdata", "olm")
+			clustercatalogTemplate = filepath.Join(baseDir, "clustercatalog.yaml")
+			clustercatalog         = olmv1util.ClusterCatalogDescription{
+				Name:     "clustercatalog-69123",
+				Imageref: "quay.io/olmqe/olmtest-operator-index:nginxolm69123",
+				Template: clustercatalogTemplate,
+			}
+		)
+		g.By("Create clustercatalog")
+		defer clustercatalog.Delete(oc)
+		clustercatalog.Create(oc)
+
+		g.By("get the index content through http service on cluster")
+		unmarshalContent, err := clustercatalog.UnmarshalContent(oc, "all")
+		o.Expect(err).NotTo(o.HaveOccurred())
+
+		allPackageName := olmv1util.ListPackagesName(unmarshalContent.Packages)
+		o.Expect(allPackageName[0]).To(o.ContainSubstring("nginx69123"))
+
+		channelData := olmv1util.GetChannelByPakcage(unmarshalContent.Channels, "nginx69123")
+		o.Expect(channelData[0].Name).To(o.ContainSubstring("candidate-v0.0"))
+
+		bundlesName := olmv1util.GetBundlesNameByPakcage(unmarshalContent.Bundles, "nginx69123")
+		o.Expect(bundlesName[0]).To(o.ContainSubstring("nginx69123.v0.0.1"))
```

Comment:
This tests is not using OLM at all.
What it is testing out? 
What is the goal of this tests?